client/components/global-notices/test/index.js

```diff
@@ -0,0 +1,84 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal Dependencies
+ */
+import { NoticesList } from '..';
+import Notice from 'components/notice';
+
+const baseProps = deepFreeze( {
+	storeNotices: [],
+	removeNotice: jest.fn(),
+
+	// Stub event-emmiter notice store so data-observe doesn't break
+	notices: { on() {}, off() {} },
+} );
+
+beforeEach( jest.clearAllMocks );
+
+describe( '<NoticesList />', () => {
+	test( 'should not render without notices', () => {
+		const wrapper = shallow( <NoticesList { ...baseProps } /> );
+		expect( wrapper.type() ).toBeNull();
+	} );
+
+	test( 'should render notices with expected HTML', () => {
```

Comment:
Super-picky, but maybe rephrase to `expected markup` or `expected DOM` or `expected JSX`? Simply because a `shallow` test doesn't really go as far as checking the actual HTML produced (as is reflected by the Jest snapshot).